### PR TITLE
Docs styling overhaul and cleanup

### PR DESCRIPTION
**Pull request by lanthias**
Thank you so much for contributing to Datapane, please help us by filing out this PR template and ensure you have read the [CONTRIBUTING.md](../CONTRIBUTING.md).

It may take a few days to review your PR and merge it - please bear with us!

Thanks!

## Prerequsites

- [x] Has been tested locally

## Proposed Changes

Our docs needed some love, as they had content which was no longer relevant, or was erroneous, including:
- Errors around workspaces and cloud (and the relevant workflows)
- Guides and content from before which didn't make sense 
- Datapane Enterprise docs, which were no longer relevant

Additionally, they didn't adhere to the same styling which we have across the rest of the application.

With this in mind, I have done the following:

## Update the styling and design 

- Using some tailwind base stylings across the docs to give consistency
- Font consistency with our main product
- New colour scheme which is compatible with our product
- Fixed dark mode styling

image

image

image

image

## Update content
- Added new copy to landing page of docs, including needs payoff for diffrentiators
- Tweaked tutorials
- Rewrote cloud sections

## Remove old content
- Removed enterprise
- Removed guides
- Removed broken references to things like `signup` and other teams features

## Add analytics and chat
- Added PostHog
- Added Intercom

This is a relatively large PR and change, but I think it will set a stronger baseline for us to build on.

### CODE REVIEW

File: docs/docs/tutorials/basics/create-an-app.ipynb [hunk 1]
```diff
@@ -122,13 +116,13 @@
...
     "\n",
-    "That app was pretty basic, but we can jazz it up by adding some plots and Markdown text. Unlike a traditional BI tool, Datapane does not rely on a proprietary visualization engine; instead, it natively supports Python visualization libraries such as [Altair](https://altair-viz.github.io/), [Plotly](https://plotly.com/python/), [Bokeh](https://bokeh.org/), and [Folium](https://python-visualization.github.io/folium/).\n",
+    "## A richer app\n",
     "\n",
-    "We also support an advanced Table block, called `DataTable`, which allows sorting, filtering, and interactive analysis - however it requires uploading your app to a Datapane server to function.\n",
+    "That app was pretty basic, but we can jazz it up by adding some plots and Markdown text. Unlike a BI tool, Datapane does not rely on a proprietary visualization engine; instead, it natively supports Python visualization libraries such as [Altair](https://altair-viz.github.io/), [Plotly](https://plotly.com/python/), [Bokeh](https://bokeh.org/), and [Folium](https://python-visualization.github.io/folium/).\n",
```
Comment (shahinrostami) [inline docs/docs/tutorials/basics/create-an-app.ipynb:125]: `[PlotAPI](https://plotapi.com), and [Plotly](https://plotly.com/python/)` ☺️

File: docs/docs/tutorials/datapane-cloud/authenticate-datapane.ipynb [hunk 2]
```diff
@@ -7,29 +7,20 @@
    "source": [
     "# Authentication\n",
     "\n",
-    "First, we need to authenticate with Datapane Cloud.\n",
+    "To host your app on Datapane Cloud, create a free API key and login using your local Datapane library.\n",
```
Comment (shahinrostami) [inline docs/docs/tutorials/datapane-cloud/authenticate-datapane.ipynb:10]: I would rephrase this: "login using your local Datapane library"

File: docs/docs/tutorials/datapane-cloud/projects.md [hunk 3]
```diff
@@ -1,6 +1,5 @@
-# Projects
-
-## Projects
+!!! info
+If you are on the Cloud Starter plan, you will be limited to a single Project, and won't be able to create another until you upgrade to Cloud Pro.
```
Comment (shahinrostami) [inline docs/docs/tutorials/datapane-cloud/projects.md:2]: Admonition is broken

Screenshot 2022-10-21 at 14 25 13

File: docs/docs/tutorials/datapane-cloud/projects.md [hunk 4]
```diff
@@ -1,6 +1,5 @@
-# Projects
```
Comment (shahinrostami) [inline docs/docs/tutorials/datapane-cloud/projects.md:1]: Some pages, like this one, have had this kind of heading removed. 

Some haven't, e.g. workspaces:

Screenshot 2022-10-21 at 14 28 06

File: docs/docs/tutorials/datapane-cloud/sharing.ipynb [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    "```\n",
+    "\n",
+    "## Workspaces\n",
+    "\n",
+    "When you created your Datapane Cloud account, you will have named your Workspace, which allows you to securely collaborate with your team. \n",
+    "\n",
+    "!!! note\n",
+    "    Every Datapane Cloud account is a member of a workspace.\n",
+    "\n",
+    "In order to securely share your reports, invite your team to your workspace in [Workspace Settings -> Members](https://cloud.datapane.com/teams-settings-team/). \n",
```
Comment (shahinrostami) [inline docs/docs/tutorials/datapane-cloud/sharing.ipynb:37]: "In order to" -> "To"

File: docs/docs/tutorials/styling/global-styling.md [hunk 6]
```diff
@@ -1,10 +1,8 @@
-# Global Styling and Whitelabelling
-
-This section describes how to add custom colours and fonts to your app.
+This section describes how to add custom colours and fonts to all the apps in your workspace.
```
[the remark this answers is not in the document]
Comment (shahinrostami) [reply docs/docs/tutorials/styling/global-styling.md:1]: "workspace" -> "Workspace" (?)

File: docs/docs/tutorials/styling/global-styling.md [hunk 7]
```diff
@@ -1,10 +1,8 @@
...
 
 ## Styles
 
-Datapane Enterprise allows you to define a global style block which will apply to all of your apps. In this block you can \(re\)define the default font, background colour and more.
+Datapane allows you to define a global style block which will apply to all of your apps. In this block you can \(re\)define the default font, background colour and more.
```
Comment (shahinrostami) [inline docs/docs/tutorials/styling/global-styling.md:5]: "colour" -> "color"

File: docs/docs/tutorials/styling/index.ipynb [hunk 8]
```diff
@@ -5,8 +5,6 @@
    "id": "e7778e85-2b10-499a-9679-77f6ba5a3ffc",
    "metadata": {},
    "source": [
-    "# Styling\n",
```
Comment (shahinrostami) [inline docs/docs/tutorials/styling/index.ipynb:8]: "colour" -> "color" through this page

**Review by shahinrostami — APPROVED** [review]
Looking good! 

I hope my review comments are useful! It's a single (but careful) pass over the changes.

The component preview images have disappeared:

Screenshot 2022-10-21 at 14 44 57

This is because we don't generate the image previews anymore (we embed apps directly). The solution is either to take a screenshot of the two components and include those, or include the components directly (now that they're lightweight!). I can do this in a branch once this one is merged if you'd prefer!

File: docs/mkdocs.yml [hunk 9]
```diff
@@ -9,31 +9,23 @@ nav:
...
-          - App visibility: "tutorials/datapane-cloud/app-visibility.ipynb"
+          - Login to Cloud: "tutorials/datapane-cloud/authenticate-datapane.ipynb"
+          - Upload your app: "tutorials/datapane-cloud/upload-an-app-to-datapane-cloud.ipynb"
+          - Share your app: "tutorials/datapane-cloud/sharing.ipynb"
           - Workspaces:
               - "tutorials/datapane-cloud/workspaces/index.md"
               - Workspace settings: "tutorials/datapane-cloud/workspaces/workspace-settings.md"
               - Managing members: "tutorials/datapane-cloud/workspaces/managing-members.md"
           - Projects: "tutorials/datapane-cloud/projects.md"
-      - 🏢 Datapane Enterprise:
```
[the remark this answers is not in the document]
Comment (lanthias) [reply docs/mkdocs.yml:27]: Good catch!

File: docs/mkdocs.yml [hunk 10]
```diff
@@ -9,31 +9,23 @@ nav:
...
-      - 🎨 Styling:
-          - tutorials/styling/index.ipynb
+      - Styling:
+          - Styling a single app: "tutorials/styling/index.ipynb"
           - Global styling: "tutorials/styling/global-styling.md"
-      - 🤖 Automation:
+      - Automation:
           - tutorials/automation/index.md
 
   - Catalogues:
```
[the remark this answers is not in the document]
Comment (shahinrostami) [reply docs/mkdocs.yml:31]: I can do this in an branch once this is merged if you prefer - will involve setting up redirects too

[the remark this answers is not in the document]
Comment (lanthias) [reply docs/mkdocs.yml:31]: That would be great, thanks.

**Comment by mands** [comment]
LGTM
